Source/Core/Core/HW/Wiimote.cpp

```diff
@@ -18,16 +18,12 @@
 #include "Core/IOS/USB/Bluetooth/BTEmu.h"
 #include "Core/IOS/USB/Bluetooth/WiimoteDevice.h"
 #include "Core/Movie.h"
-#include "Core/NetPlayClient.h"
 #include "Core/System.h"
 #include "Core/WiiUtils.h"
 
 #include "InputCommon/ControllerEmu/ControlGroup/ControlGroup.h"
 #include "InputCommon/InputConfig.h"
 
-// Limit the amount of wiimote connect requests, when a button is pressed in disconnected state
-static std::array<u8, MAX_BBMOTES> s_last_connect_request_counter;
```

Comment:
This code (which seems to have always been mostly removed in 13ee6bd7bb50d0f23d32383f396e4199533868cd/#8985) *might* be useful since games will immediately disconnect the balance board if they don't want it (or wiimote 4 when they use the balance board) IIRC. But it's probably not too important as long as the bindings are distinct for the balance board and wii remote (I remember in early testing, I had both balance board power and wiimote 1 A on the same key).

Reply:
Yeah, it's a remnant from long ago cleanups. There's a `m_connection_request_counter` in the `WiimoteDevice` class now.